scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java

```diff
@@ -70,7 +70,7 @@ private static Problem makeTestMissionAB() {
     return new Problem(fooMissionModel, h, new SimulationFacade(h, fooMissionModel), SimulationUtility.getFooSchedulerModel());
   }
 
-  private final static PlanningHorizon h = new PlanningHorizon(TimeUtility.fromDOY("2025-001T01:01:01.001"), TimeUtility.fromDOY("2030-001T01:01:01.001"));
+  private final static PlanningHorizon h = new PlanningHorizon(TimeUtility.fromDOY("2025-001T01:01:01.001"), TimeUtility.fromDOY("2025-005T01:01:01.001"));
```

Comment:
Because simulating 5 years of mission model is too much in terms of computing time. When scheduling starts, it needs to simulate the whole planning horizon (for evaluating coexistence expressions for example).

Reply:
Yes, but isn't running scheduling over a plan multiple years long a use case of ours? If so, we need to be testing how the scheduler handles that. Is there another test that runs covers this?

Reply:
Performance is definitely a concern but I am not sure we should have to wait for a long time every time we run unit tests.

Reply:
My concern is less with performance and more that if we never test the scheduler against a duration greater than a few days, then we can end up missing bugs/issues that only appear on longer plans.

Reply:
Did this test recently start to take a longer time to run? Or has it always been taking a long time? Also - how long is a long time?

Reply:
Yes, it started to take a much longer time as a consequence of this fix because the timeline is now accurately go forward in time. A 5-year simulation is a lot. I am not sure I did see the end of it.

Reply:
I have made one change concerning returning different simulation results if the start timestamp is different. And added tests about that. And added a 5-year test with Banananation.